docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+shipit:
+  build: ./
+  command: python kickoff-web.py -d sqlite:///update.db --username=admin --password=password --host=0.0.0.0 --port=5000 --verbose
```

Comment:
Ah, you're right, I missed that. In any case....using MySQL is much better to more closely match production.